app/jobs/seed_users_data_job.rb

```diff
@@ -89,7 +89,7 @@ def per_patient_traits
     @per_patient_traits ||=
       {
         ongoing_bp: {
```

Comment:
What is the difference between ongoing and retroactive here in terms of how seed data is created?

Reply:
At first glance, I thought that `ongoing` creates BPs for patients that are "regularly returning to care", and `retroactive` creates BPs for patients that have "stopped coming back to care". I just noticed that we're omitting `recorded_at` from the latter, so there may be more going on.

Reply:
On second thought, I think removing the `recorded_at` is maybe not the best thing? I'm not sure.

Basically if you provide a `recorded_at`, it will set the patient registration date to be that if the BP is backdated (thereby modifying the patient trait that you are intentionally creating at certain dates)

If you don't provide a `recorded_at`, it will take the `device_created_at` anyway and modify it regardless.

I think we can just remove that exclusion, just adds confusion.

Since the app now always sends a recorded app for BPs anyway, it's also more realistic to not remove it and let the shape of the generated data take whatever form it needs to.